### PR TITLE
Minor UI changes to search bar colors and focus

### PR DESCRIPTION
**Pull request by jesstn**
Fix changes issue #24

### CODE REVIEW

File: app/units/units.css [hunk 1]
```diff
@@ -45,7 +46,7 @@ input:focus, input:first-line {
 .unitSearchInput:invalid{ background-color:transparent; }
 
 .unitSearch .unitSearchInput:focus {
-    border: 2px solid red;
+    outline: none;
```
Comment (GoodGameRuler) [inline app/units/units.css:49]: Looks good but would recommend some form of feedback for user.

was thinking something like a thicker line or maybe an accent colour

```css
.unitSearch .unitSearchInput:focus {
    outline: none;
    border: 3px solid #DD432B;
}
```

**Review by GoodGameRuler — CHANGES_REQUESTED** [review]
Request changes for input focus, but open to not include if it doesn't look. Will leave it to you.

File: app/units/units.css (hunk 1 above)
Comment (jesstn) [reply app/units/units.css:49]: Good catch, for feedback I deleted the border and changed the search bar colour.

Comment (jennifermtan) [reply app/units/units.css:49]: I like this UI change, thanks!

Comment (GoodGameRuler) [reply app/units/units.css:49]: Good Change! Resolving this Convo.

**Review by GoodGameRuler — APPROVED** [review]

**Pull request merged**